llm-proxy/litellm-config.yaml

```diff
@@ -6,6 +6,19 @@ model_list:
 # -------------------
 
 # Gemini models
+
+- model_name: gemini-pro
```

Comment:
This needs to be renamed to Gemini-1.5-pro

Reply:
done